Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -725,8 +738,8 @@ class DropDownMenu
         event.stopPropagation();
         if (dropDownMenu !== DropDownMenu._currentMenu)
         {
-          DropDownMenu.OnClick (dropDownMenu, dropDownMenu[0].id, getSelectionCount, null);
-          event.keyCode = 40; // always act as if the down-arrow was used when opening the drop down menu.
+          DropDownMenu.OnClick (dropDownMenu, dropDownMenu.id, getSelectionCount, null);
+          (event.keyCode as any) = 40; // always act as if the down-arrow was used when opening the drop down menu.
```

Comment:
wouldn't this be number?

Reply:
Yes. changed. FYI it's a deprecated API: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
Could you please make a ticket for this (don't need a todo). We need to schedule the cleanup before the API goes the way of the Dodo.

@iam-mholle , @a-ctor We also use this API in the support-versions, do we have any indication when we might expect the API to die so we know when/if we need to fix the support branches and roll out an update?

Reply:
It been deprecated for a long time, the `code` property should be used instead. I don't think that the property is ever going to be removed, but any browser might remove it at any time

Reply:
Then we just try to update this sometime this year and do not do a backport.

Reply:
So this does not get changed in this PR? Is there an existing issue or should I create one?

Reply:
@iam-mholle correct, we keep keyCode in this PR. Please create a ticket.

Reply:
Added https://re-motion.atlassian.net/browse/RM-7733